synfig-studio/src/gui/app.cpp

```diff
@@ -1433,14 +1483,15 @@ App::App(const synfig::String& basepath, int *argc, char ***argv):
 	Glib::set_application_name(_("Synfig Studio"));
 
 	Splash splash_screen;
-	splash_screen.show();
+	// Is it safe? I don't know, but that's how I got Splash Screen back
+	splash_screen.show_now();
```

Comment:
`splash_screen.show();` works fine for me. Both on Linux and Windows. 
Did you have any issues here?

Reply:
Yes, no Splash Screen appears.
But maybe it was something memory cached and opened it fast enough to "hide" it

Reply:
Yes, it's working now o.O And I fought against it for 30min ¬¬